src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -236,5 +237,7 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
 
             return bestItems.ToImmutableAndFree().SelectAsArray(itemWithPatternMatch => itemWithPatternMatch.Item1);
         }
+
+        internal virtual ImmutableArray<CompletionProvider> GetCompletionProviders() => ImmutableArray<CompletionProvider>.Empty;
```

Comment:
what is going on here?

Reply:
Needed a way to get the built in completion providers since they weren't exported.  But now that method is obsolete so I need a different way.